spec/request/user_request_spec.rb

```diff
@@ -95,4 +95,73 @@
       )
     end
   end
+
+  context 'when an LOA3 user is logged in', :skip_mvi do
+    let(:loa3_user) { build(:user, :loa3) }
+    let(:auth_header) { { 'Authorization' => "Token token=#{token}" } }
+
+    before do
+      Session.create(uuid: loa3_user.uuid, token: token)
+      User.create(loa3_user)
+      create(:account, idme_uuid: loa3_user.uuid)
+    end
+
+    it 'GET /v0/user - for MVI error should only make a request to MVI one time per request!' do
```

Comment:
This fails 
````
   it 'GET /v0/user - for MVI error should only make a request to MVI one time per request!' do
      stub_mvi_failure
      expect_any_instance_of(Common::Client::Base).to receive(:perform).twice.and_call_original
      get v0_user_url, nil, auth_header
      get v0_user_url, nil, auth_header
    end
````

Reply:
never mind.  @saneshark explained that `expect_any_instance_of` doesn't work like that.